### DISCUSSION TITLE
Error when running the game with Obfuscated JAR

### DISCUSSION THREAD

**Discussion by sophia-yxfff** [CastleLab/COMP3021-2021Fall-PA1-Student-Version#49] [Q&A] [answered]
I got the follow errors when I try to run the game with Obfuscated JAR (my command line is  java --enable-preview -jar PA1-obf.jar [--unicode] puzzles/00-unicode-test.game):
Error: LinkageError occurred while loading main class hk.ust.cse.comp3021.pa1.Main
        java.lang.UnsupportedClassVersionError: hk/ust/cse/comp3021/pa1/Main has been compiled by a more recent version of the Java Runtime (class file version 60.65535), this version of the Java Runtime only recognizes class file versions up to 59.0

<img width="1391" alt="Screenshot 2021-10-03 at 12 08 29 AM" src="https://user-images.githubusercontent.com/57562370/135724208-da3b6d02-40ff-457f-b394-b245b2a76313.png">

May I know how to fix it?
Many thanks!

**Comment by Derppening** [comment, marked as answer]
You are using Java 15. Please update to Java 16.

**Reply by sophia-yxfff** [reply]
Thank you! It was fixed
